packages/core/useModifierState/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ref } from 'vue-demi'
+import { useEventListener } from '../useEventListener'
+
+export function useModifierState(modifier: string) {
```

Comment:
I see the doc states only a few of possible options, can we better type them as literals?